src/hotspot/share/gc/g1/g1CollectedHeap.cpp

```diff
@@ -3312,98 +3312,72 @@ class G1STWDrainQueueClosure: public VoidClosure {
   }
 };
 
-// Parallel Reference Processing closures
-
-// Implementation of AbstractRefProcTaskExecutor for parallel reference
-// processing during G1 evacuation pauses.
-
-class G1STWRefProcTaskExecutor: public AbstractRefProcTaskExecutor {
-private:
-  G1CollectedHeap*          _g1h;
-  G1ParScanThreadStateSet*  _pss;
-  G1ScannerTasksQueueSet*   _queues;
-  WorkGang*                 _workers;
+class G1STWRefProcClosureContext : public AbstractClosureContext {
+  uint                           _max_workers;
+  uint                           _queues;
```

Comment:
Please avoid lining up member names. That is hard to read. Group them by functionality/contents.

Reply:
Agree, will remove all spaces.